source/_integrations/todoist.markdown

```diff
@@ -64,7 +64,7 @@ Creating custom projects is super-easy and quite powerful. All you need to run t
 # Example configuration.yaml entry
 calendar:
   - platform: todoist
-    token: YOUR_API_TOKEN
+    token: !secret todoist_api_token
```

Comment:
We should remove the secret note below instead and keep the constant here.

Reply:
Done